app/assets/javascripts/govuk_publishing_components/components/input.js

```diff
@@ -0,0 +1,48 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function Input () { }
+
+  Input.prototype.start = function ($module) {
+    this.$module = $module[0]
+    this.input = this.$module.querySelectorAll('.gem-c-input')[0]
```

Comment:
Good point. I actually thought this as I wrote it, but for some reason didn't follow through. Will change.

Reply:
Changed.

Reply:
I was going to suggest a slightly different solution - which is to remove the `[0]` and loop through resulting `NodeList`. That way this can handle either single or multiple password inputs on the same page.

Reply:
But because this is a component the JS will still be initialised for each instance of it on the page. I don't want to start trying to do a single initialisation for all where I have to check if it's already been done. And the most we'll have it on any page is twice.